app/views/contexts/summary/list_collector_block.py

```diff
@@ -151,18 +95,6 @@ def get_repeating_block_related_answer_blocks(
                 blocks.extend(answer_blocks)
```

Comment:
I am not sure about this one, this method is only used in context of generating display of calculated summary and grand calculated summary, so we should only move it to base class if content list collector supports calculated summary?

Reply:
On my[ supplementary data branch ](https://github.com/ONSdigital/eq-questionnaire-runner/pull/1173) I've got a schema which uses repeating blocks to ask questions about a supplementary list and then it has a calculated summary of those answers, so I think we'll want the content block to support calculated/grand calculated summaries too

Reply:
Just for additional info/context, list collector content block needs to support everything that a list collector can do bar add, edit and remove and that it is a content block instead of a question. Other than that, it should behave identically. Hence, the importance to keep code shared as much as possible.

Reply:
Ok, added now. Thanks.